definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-share-a-case-nonprod.json

```diff
@@ -5,5 +5,19 @@
     "CaseStateID": "SOTAgreementPayAndSubmitRequired",
     "UserRole": "caseworker-caa",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "30/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "Submitted",
+    "UserRole": "caseworker-caa",
+    "CRUD": "CRU"
```

Comment:
Little thing - CAA only ever requires R access to states, but on a wider note, CAA users aren't in Share A Case at all - so might be better putting anything CAA related into a new toggle file

Reply:
Well, apparently they are. Without these permissions it was not working.